public/index.html

```diff
@@ -43,7 +43,11 @@
     />
 
     <script src="https://use.fontawesome.com/733e69044c.js"></script>
-    <link href="https://fonts.googleapis.com/css?family=Rubik:wght@500;600;700;800&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
```

Comment:
What does this do? Improve the fonts load time?

Reply:
in theory, maybe; in practice, not really